src/mscorlib/corefx/System/Security/CryptographicException.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Runtime.Serialization;
+
+namespace System.Security.Cryptography
+{
+    [Serializable]
+    public class CryptographicException : SystemException
```

Comment:
So are we now exposing CryptographicException from coreclr as well?  Presumably it needs to be added to model.xml as well then?